### ISSUE TITLE
[Feature Request]: Dying on foreign planets is too punishing

### ISSUE THREAD

**Issue by Blayung** [terrarium-earth/Ad-Astra#602]
### Mod Version

1.15.18

### Mod Loader Version

1.20.1 - 0.15.11

### Mod Loader

Fabric

### Additional context

Basically, dying on e.g. the moon is so shit, you have to waste like another 3 stacks of iron for another rocket and a suit. I do not know what would be a suitable solution to this, but you should really do something about it.

**Comment by Dorfyt** [comment]
Skill issue

**Comment by Blayung** [comment]
@EnderDrag0n Yeah, I haven't actually died even once there. It's just a random thought I had :)
